pom.xml

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.4.4</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>org.folio</groupId>
+  <artifactId>edge-inn-reach</artifactId>
+  <name>edge-inn-reach</name>
+  <version>1.0.0-SNAPSHOT</version>
+  <description>FOLIO edge-inn-reach module (Spring based implementation)</description>
+  <packaging>jar</packaging>
+
+  <licenses>
+    <license>
+      <name>Apache License 2.0</name>
+      <url>http://spdx.org/licenses/Apache-2.0</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <java.version>11</java.version>
+    <folio-spring-base.version>1.0.5</folio-spring-base.version>
+    <openapi-generator.version>5.1.0</openapi-generator.version>
+    <mapstruct.version>1.4.1.Final</mapstruct.version>
+    <wiremock-version>2.27.2</wiremock-version>
+    <json.version>20210307</json.version>
+    <jackson.version>2.12.2</jackson.version>
+    <junit.version>5.7.0</junit.version>
+    <rest-assured.version>4.3.3</rest-assured.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- Api definition files -->
+    <i2nreach.yaml.file>
+      ${project.basedir}/src/main/resources/swagger.api/i2nreach.yaml
+    </i2nreach.yaml.file>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.junit</groupId>
+        <artifactId>junit-bom</artifactId>
+        <version>${junit.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <type>pom</type>
+        <scope>import</scope>
+        <version>${jackson.version}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
```

Comment:
Please add actuator for health check, all modules which are follow spring way uses actuator
```
    <dependency>
      <groupId>org.springframework.boot</groupId>
      <artifactId>spring-boot-starter-actuator</artifactId>
    </dependency>
```

Reply:
Done